package.json

```
@@ -24,38 +24,38 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^5.0.0",
+    "@typescript-eslint/experimental-utils": "^5.57.0",
```

should really migrate to the `utils` package
 
```suggestion
    "@typescript-eslint/utils": "^5.57.0",
```

---

Oh I did not know there is stable version now.
Do you know if there were any breaking changes between experimental and stable versions?

---

Nope no changes - `experimental-utils` is now just an alias for `utils`!